telos-frontend/src/dnd/DraggableWidget.jsx

```diff
@@ -13,8 +14,8 @@ function getStyles(left, top) {
 }
 
 // A component for widgets that can be dragged
-const DraggableWidget = memo((props) => {
-  const { id, position, type, data, date } = props;
+const DraggableWidget = memo(({ id, position, type, data, date, deleteWidget }) => {
+  // const { id, position, type, data, date, update } = props;
```

Comment:
Could delete this commented line since you refactored the code?

Reply:
Oh yep i'll quickly remove it now